ql/src/java/org/apache/hadoop/hive/ql/parse/AlterTableExecuteSpec.java

```diff
@@ -146,13 +157,19 @@ public boolean isExpireByTimestampRange() {
       return timestampMillis != -1 && fromTimestampMillis != -1;
     }
 
+    public boolean isExpireByRetainLast() {
+      return numRetainLast != -1;
```

Comment:
What happened if `numRetainLast` <-1, e.g. -2
We can check this value must be >= 1 in somewhere.

Reply:
There is a check here:
https://github.com/apache/iceberg/blob/main/core/src/main/java/org/apache/iceberg/RemoveSnapshots.java#L134-L137

Reply:
shouldn't it be
````
return numSnapshots >= 1
````

Reply:
discussed offline, we are sorted